BDKSwiftExampleWallet/Model/BackupInfo.swift

```diff
@@ -12,7 +12,7 @@ struct BackupInfo: Codable, Equatable {
     var descriptor: String
     var changeDescriptor: String
 
-    init(mnemonic: String, descriptor: String, changeDescriptor: String) {
+    init(mnemonic: String = "", descriptor: String, changeDescriptor: String) {
```

Comment:
is there any reason for this default empty string being here vs the way it was before?